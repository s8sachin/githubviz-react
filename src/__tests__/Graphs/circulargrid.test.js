import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Circulargridcharts from '../../components/Graphs/circulargridlinechart';

configure({ adapter: new Adapter() });

describe('Circulargridcharts', () => {
it('should mount circulargrid component', () => {
    const wrapper = shallow(<Circulargridcharts />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})