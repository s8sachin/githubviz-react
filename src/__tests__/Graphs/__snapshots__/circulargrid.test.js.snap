// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circulargridcharts should mount Header component 1`] = `
<XYPlot
  className=""
  height={300}
  margin={
    Object {
      "bottom": 10,
      "left": 10,
      "right": 10,
      "top": 10,
    }
  }
  width={300}
  xDomain={
    Array [
      -3,
      3,
    ]
  }
  yDomain={
    Array [
      -3,
      3,
    ]
  }
>
  <CircularGridLines
    centerX={0}
    centerY={0}
  />
  <XAxis
    attr="x"
    attrAxis="y"
    orientation="bottom"
    top={145}
  />
  <YAxis
    attr="y"
    attrAxis="x"
    left={140}
    orientation="left"
  />
  <MarkSeries
    data={
      Array [
        Object {
          "r": 1,
          "size": 30,
          "theta": 1.0471975511965976,
          "x": 0.5000000000000001,
          "y": 0.8660254037844386,
        },
        Object {
          "r": 1.7,
          "size": 10,
          "theta": 2.0943951023931953,
          "x": -0.8499999999999996,
          "y": 1.4722431864335457,
        },
        Object {
          "r": 2,
          "size": 1,
          "theta": 3.141592653589793,
          "x": -2,
          "y": 2.4492935982947064e-16,
        },
        Object {
          "r": 3,
          "size": 12,
          "theta": 4.71238898038469,
          "x": -5.51091059616309e-16,
          "y": -3,
        },
        Object {
          "r": 2.5,
          "size": 4,
          "theta": 0.7853981633974483,
          "x": 1.7677669529663689,
          "y": 1.7677669529663687,
        },
        Object {
          "r": 0,
          "size": 1,
          "theta": 0.7853981633974483,
          "x": 0,
          "y": 0,
        },
      ]
    }
    getNull={[Function]}
    sizeRange={
      Array [
        5,
        15,
      ]
    }
    strokeWidth={2}
  />
</XYPlot>
`;
